it/mail/chrome/messenger/messenger.properties

```diff
@@ -140,46 +140,46 @@ messageCollapsed=Raggruppato
 
 # Used in the SMTP Account Settings panel when a server value has no properties
 smtpServerList-NotSpecified=<non specificato>
-smtpServer-ConnectionSecurityType-0=None
-smtpServer-ConnectionSecurityType-1=STARTTLS, if available
+smtpServer-ConnectionSecurityType-0=Nessuno
+smtpServer-ConnectionSecurityType-1=STARTTLS, se disponibile
 smtpServer-ConnectionSecurityType-2=STARTTLS
 smtpServer-ConnectionSecurityType-3=SSL/TLS
 smtpServer-SecureAuthentication-Type-false=No
-smtpServer-SecureAuthentication-Type-true=Si
+smtpServer-SecureAuthentication-Type-true=Sì
 smtpServers-confirmServerDeletionTitle=Elimina server
-smtpServers-confirmServerDeletion=Eliminare il server:\n%S
+smtpServers-confirmServerDeletion=Sei sicuro di volere eliminare il server:\n%S
```

Comment:
CORRECTION NEEDED. 

If this is a question requiring the user's confirmation, a "?" is needed at the end of the sentence.